test/Wiki.t.sol

```diff
@@ -46,7 +79,12 @@ contract TestWiki is Test {
         string memory ipfs = "Qmb7Kc2r7oH6ff5VdvV97ynuv9uVNXPVppjiMvkGF98F6v";
         uint256 _deadline = block.timestamp;
 
-        (uint8 v, bytes32 r, bytes32 s) = sign(privateKey, ipfs, editor, _deadline);
+        (uint8 v, bytes32 r, bytes32 s) = sign(
```

Comment:
pls lets make sure we dont add noise to Pull Request from styles that I dont think have been defined. Contracts its something we need to be very careful, and adding 40 loc to review of styling its a pretty bad practice

Reply:
sorry about this I think this was a result of my linter I didn't actually style it myself